frontend/src/scenes/surveys/Surveys.tsx

```diff
@@ -60,7 +60,9 @@ export function Surveys(): JSX.Element {
 
     const { user } = useValues(userLogic)
     const { currentTeam } = useValues(teamLogic)
-    const surveysPopupDisabled = currentTeam && !currentTeam?.surveys_opt_in
+    const { updateCurrentTeam } = useActions(teamLogic)
+    const surveysPopupDisabled =
+        currentTeam && !currentTeam?.surveys_opt_in && surveys.some((s) => s.start_date && !s.end_date)
```

Comment:
should we check to make sure the surveys they have active actually use the popover instead of just the API? If they use only api surveys it'd be annoying to see this all the time, but intentionally have it disabled.

Reply:
API surveys case now accounted for